srcs/app/server/Server.cpp

```diff
@@ -114,7 +114,7 @@ void	Server::BindListeningSocket_() const {
 }
 
 void	Server::AddConnection_(int sd) {
-	HttpRequest *request = new HttpRequest();
+	HttpRequest *request = new HttpRequest(sd);
```

Comment:
I don't really like this because the `HttpRequest` does not need the socket. But to set the `REMOTE_ADDR` env variable in the `CGI` class, we have to pass the socket or the address as a string somehow.

I couldn't think of a better solution. The alternative was to pass the socket to the HttpResponseFactory->Http*Response->CGI.